### ISSUE TITLE
Failed: nightly against akka master failed because assertAllStagesStopped is not compatible with master any more

### ISSUE THREAD

**Issue by jrudolph** [akka/akka-http#2421]
E.g. https://jenkins.akka.io:8498/job/akka-http-nightly/388/AKKA_VERSION=master,Node=akka-http-nightly-node,SCALA_VERSION=2.12.8,jdk=AdoptOpenJDK%208/console

Compilation error:

```
[error] /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/master/Node/akka-http-nightly-node/SCALA_VERSION/2.12.8/jdk/AdoptOpenJDK 8/akka-http-core/src/test/scala/akka/stream/testkit/Utils.scala:46:49: value PrintDebugDump is not a member of object akka.stream.impl.StreamSupervisor
[error] Error occurred in an application involving default arguments.
[error]           children.foreach(_ ! StreamSupervisor.PrintDebugDump)
[error]                                                 ^
[error] one error found
```

**Comment by jrudolph** [comment]
Only affects tests. Probably not fixable until we update to the latest (yet-unreleased) Akka version.

**Comment by jrudolph** [comment]
Fixes by #2427.
